Tests/Smoketest4.xctestplan

```diff
@@ -61,6 +61,7 @@
         "SaveLoginTest\/testDeleteLogin()",
         "SaveLoginTest\/testDoNotSaveLogin()",
         "SaveLoginTest\/testLoginsListFromBrowserTabMenu()",
+        "SaveLoginTest\/testSaveLogin()",
```

Comment:
@isabelrios `testSaveLogin()` fails only on Bitrise. I haven't figured out the cause. The test website is loaded properly with the username and the password filled, but the username isn't saved.

Should we just disable the test for now?

Reply:
That test is passing consistently on Bitrise for all PRs and merges in main. I would not disable it but undo the changes related to it on this PR so that it keeps passing.

I know it was showing differently in M1 and Intel on automation, that's why we had the condition to show 1 or 2 logins. For some reason on Intel the username is not detected (oas far as I know only on automation, not a bug) and so it assumes it is the same login and so there is only one created. 

As we know this behaviour, I would live with it instead of disabling the test and reducing the coverage in this feature.

Reply:
🆗 Let me undo the change near line 93.